Dockerfile

```diff
@@ -18,9 +18,8 @@ ENV EXTERNAL_SCRIPTS=hubot-diagnostics,hubot-help,hubot-google-images,hubot-goog
 RUN yo hubot --owner="$BOT_OWNER" --name="$BOT_NAME" --description="$BOT_DESC" --defaults && \
 	sed -i /heroku/d ./external-scripts.json && \
 	sed -i /redis-brain/d ./external-scripts.json && \
-	npm install hubot-scripts
-
-ADD . /home/hubot/node_modules/hubot-rocketchat
+	npm install hubot-scripts && \
+	npm install hubot-rocketchat
```

Comment:
I had to install hubot-rocketchat that way, otherwise it would keep on complaining that it does not exist. Maybe I'm doing something wrong in the build process